src/main/java/uk/gov/pay/directdebit/payers/model/Payer.java

```diff
@@ -0,0 +1,114 @@
+package uk.gov.pay.directdebit.payers.model;
+
+import uk.gov.pay.directdebit.common.util.RandomIdGenerator;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+public class Payer {
+    private Long id;
+    private Long paymentRequestId;
+    private String externalId;
+    private String name;
+    private String email;
+    private String sortCode;
+    private String accountNumberLastTwoDigits;
+    private boolean accountRequiresAuthorisation;
+    private String accountNumber;
+    private String addressLine1;
```

Comment:
Jdbi does not support nested classes (yet) https://github.com/jdbi/jdbi/pull/951, that means that an inner class would have to be mapped manually - and I think that would be more hassle than the benefit it brings. Once we get the new jdbi version, though, it will be something we should definitely do 👍